README.md

```diff
@@ -281,6 +289,48 @@ SpeedScope is a web viewer for performance profiles and can be downloaded from:
 
 _Profiler_ can generate flame graphs that can be viewed using _SpeedScope_.
 
+## Understanding Semantic Versioning (MAJOR.MINOR.PATCH)
+By following `major.minor.patch`, developers communicate clearly **how risky an upgrade is** and what kind of changes users can expect.
+Version numbers in GitHub projects (and most software) usually follow **Semantic Versioning (SemVer)**.  
+A version looks like this:
+
+### 1. Major Version (`MAJOR`)
+- **Format:** `X.0.0`
+- **When to increment:**  
+  - You make **breaking changes** to the connector API or behavior.  
```

Comment:
Heeft eigenlijk alleen te maken met HelloID. Dus; als er in de code iets wijzigt waardoor er in HelloID iets kapot gaat (voorbeeld: verwijderen van de permissions.displayName), dan gaat het major omhoog. Een ander voorbeeld: als door wijziging in code de fieldMapping of configuration moet worden aangepast. Als er richting de API iets wijzigt waardoor er wel een code aanpassing nodig is, maar HelloID gewoon blijft werken, gaat het major niet omhoog.

Handig om dit soort voorbeelden er nog bij te zetten.